Source/JavaScriptCore/inspector/remote/cocoa/RemoteConnectionToTargetCocoa.mm

```diff
@@ -225,6 +228,9 @@ static void RemoteTargetHandleRunSourceWithInfo(void* info)
 
 void RemoteConnectionToTarget::sendMessageToTarget(NSString *message)
 {
+    if (m_closed)
+        return;
```

Comment:
we should also add these bailouts in `Source/JavaScriptCore/inspector/remote/RemoteConnectionToTarget.cpp`